plane/source/docs/tailsitter-tuning-guide.rst

```diff
@@ -0,0 +1,45 @@
+
+- Normally, the vehicle is stable enough with the default PIDs to do a first test hover in QSTABILIZE or QHOVER. Do a short hover flight with logging enabled such that the :ref:`Harmonic Notch Filters<common-imu-notch-filtering>` can be setup. Its important to remove as much motor noise as possible in order to get a good tune using a throttle based harmonic notch filter.
+- Once the notch filters are set, do another hover flight but briefly do short, but sharp pitch stick movements in both directions. Do not endanger the vehicle, but try to get several short full stick movements to the front and to the back. Do the same for yaw, but you should be able to hold full yaw stick each direction for a second or so, returning to neutral for a second in-between direction reversals.
+- Download the :ref:`dataflash log<common-downloading-and-analyzing-data-logs-in-mission-planner>` for analysis. In order to determine the :ref:`Q_A_RAT_PIT_FF<Q_A_RAT_PIT_FF>` and :ref:`Q_A_RAT_YAW_FF<Q_A_RAT_YAW_FF>` terms, use a log analyzer like https://plot.ardupilot.org and setup the following plots on the same scale, using the "Add Expression" button:
+
+.. code:: bash
+
+   PIQP_P+PIQP_P+PIQP_D 
+   X * (PIQP_Act) , where X is 0.2 to start
+
+- Next adjust the X value above until the magnitudes of both plots are about equal. This will now be value for the :ref:`Q_A_RAT_PIT_FF<Q_A_RAT_PIT_FF>` term. Set the :ref:`Q_A_RAT_PIT_I<Q_A_RAT_PIT_I>` term to equal this.
+- Now do the same for the YAW axis using:
+
+.. code:: bash
+
+   PIQY_P+PIQY_P+PIQY_D   
+   X * (PIQP_Act) , where X is 0.2 to start
+
+Below is an image of this where the X value is adjusted until the two curves are the same amplitude, with a value of x = 0.1. This is then set as the :ref:`Q_A_RAT_YAW_FF<Q_A_RAT_YAW_FF>` and :ref:`Q_A_RAT_YAW_I<Q_A_RAT_YAW_I>` values.
+
+.. image:: ../../../images/FF-calculation.png
+   :target: ../_images/FF-calculation.png
+
+- Now you can hover again, and begin increasing the :ref:`Q_A_RAT_PIT_D<Q_A_RAT_PIT_D>` term, either iteratively, or using :ref:`common-transmitter-tuning`, until it oscillates and then reduce it to 1/2 to 1/3 that value.
+- Then increase the :ref:`Q_A_RAT_PIT_P<Q_A_RAT_PIT_P>` term,, until it oscillates and then reduce it  1/2 to 1/3 that value.
+- Do the same for the YAW axis.
+
+This should allow you to get a reasonable tune for Pitch and Yaw. Roll is tuned like an :ref:`STL or Tilt-Rotor Quadplane <quadplane-vtol-tuning>`.
+
+Small Vehicles
+==============
+
+Smaller , more agile, Tailsitters (1m or less wingspan, and/or < 1Kg) often will benefit from raising the system's Gyro and PID Dterm filters. Increasing the :ref:`INS_GYRO_FILTER<INS_GYRO_FILTER>` to 40Hz and setting the ``Q_A_RAT_xxx_FLTD`` values to 20Hz for Pitch, Roll, and Yaw may result in a better tune, at the risk of allowing more noise into the control loops.
```

Comment:
I would be tempted to just point to the setup graphs for this bit. 

https://ardupilot.org/plane/docs/quadplane-vtol-tuning-process.html#step-3-pid-controller-initial-setup

Probably also worth mention the `SMAX` params, for small stuff your probbibly better turning them off, at least while tunning. `SMAX` will mask the effect to too high D, so you may never see the oscillation your looking for.